bin/ruby-build

```diff
@@ -918,6 +919,28 @@ use_homebrew_yaml() {
   fi
 }
 
+use_freebsd_pkg() {
+  # check if FreeBSD
+  if [ "FreeBSD" = "$(uname -s)" ]; then
+      # use openssl if installed from Ports Collection
+      if [ -f /usr/local/include/openssl/ssl.h ]; then
+	  package_option ruby configure --with-openssl-dir="/usr/local"
+      fi
+
+      # check if 11-R or later
+      if [ "$(uname -r | sed 's/[^[:digit:]].*//')" -ge 11 ]; then
+	  if $(pkg info -e libedit); then
```

Comment:
If you're only interested in the exit status of `pkg info -e` but want to discard its output, you should simply redirect to null:

```sh
if pkg info -e libedit >/dev/null; then
```